molgenis-jobs/src/main/java/org/molgenis/data/jobs/ProgressImpl.java

```diff
@@ -47,6 +48,7 @@ public void start()
 		JobExecutionContext.set(jobExecution);
 		JOB_EXECUTION_LOG.info("Execution started.");
 		jobExecution.setStartDate(Instant.now());
+		jobExecution.setProgressInt(0);
```

Comment:
Not entirely sure if this won't break the indeterminate progress bars for jobs that don't set progress info.